docs/integrations/common-workflow-steps/build-cache/how-to-share-file-between-pipelines.md

```diff
@@ -0,0 +1,44 @@
+---
+title: How to Share Files between Pipelines 
+metaTitle: How to Share Files between Pipelines
+metaDescription: How to Share Files between Pipelines
+sidebar_position: 3
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# How to Share Files between Pipelines
+
+With the build cache structure offered by Appcircle, you can share your cache files between pipelines in different branches. This file sharing will allow you to generate your packages faster in different branches and reduce your build time. Below is a simple example of how you can do this step by step. 
```

Comment:
It may be better to use passive sentences:
```suggestion
With the build cache structure provided by Appcircle, you can share cache files between pipelines in different branches. This sharing enables faster package generation across various branches, reducing overall build time. Below is a simple step-by-step example of how you can achieve this.```